h5p-bildetema/src/components/SearchPage/useSearchResults.tsx

```diff
@@ -287,7 +290,8 @@ export const useSearchResults = ({
     const filteredRes = filterSearchResults(res, filter);
 
     const visibleSearchResults = newWordsToSearchResult(
-      filteredRes.slice(0, resultAmount),
+      // filteredRes.slice(0, resultAmount),
```

Comment:
Could remove these and perhaps resultAmount and ActionType.LOAD_MORE if we don't need it 🤔

Reply:
Good point. Removed unused code